build-contract

```diff
@@ -32,12 +32,12 @@ function wait_for_contract {
 cd $ROOT/build-contracts/
 for compose_file in $(ls | grep .yml); do
   docker_compose="docker-compose -f $compose_file"
+  $docker_compose rm -f
```

Comment:
Maybe we could just remove this altogether? As we're doing `--force-recreate` anyways?